css/components/leaflet.css

```diff
@@ -55,6 +55,10 @@ img.leaflet-marker-icon.leaflet-zoom-animated.leaflet-interactive {
     display: inline-block;
     font-family: var(--font-awesome);
     padding: 0 0.3rem 0 0;
+    color: var(--color-black);
+}
+.facet-filter-visibility:before {
```

Comment:
This PR looks good to me. 

Just in terms of coding standards, `:before` and `:after` are supposed to have 2 colons `::before` and `::after` as they are pseudo "elements". 

`:focus` and `:hover` and things like that get one colon as they are pseudo classes.

===
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.